src/video.cpp

```diff
@@ -759,33 +729,22 @@ point current_resolution()
 	return point(window->get_size()); // Convert from plain SDL_Point
 }
 
-bool is_fullscreen()
```

Comment:
You removed the definition but not the declaration. Was this not useful anymore?

Reply:
SDL_WINDOW_FULLSCREEN_DESKTOP has been removed and
> you can call SDL_GetWindowFullscreenMode() to see whether an exclusive fullscreen mode will be used or the borderless fullscreen desktop mode will be used when the window is fullscreen.

doesn't seem useful since IIRC we don't do exclusive fullscreen anymore anyway.